hdk/cl/examples/cl_dram_dma/verif/scripts/Makefile.vcs

```diff
@@ -38,6 +38,6 @@ else
 endif
 
 $(COMPLIB_DIR):
-	cd $(SIM_ROOT) && echo "compile_simlib -language all -dir $(COMPLIB_DIR) -simulator $(SIMULATOR) -library all -family  all" > create_libs.tcl
+	cd $(SIM_ROOT) && echo "compile_simlib -language all -dir $(COMPLIB_DIR) -simulator $(SIMULATOR) -no_systemc_compile -library all -family  all" > create_libs.tcl
```

Comment:
Does hdk/cl/developer_designs/cl_firesim/verif/scripts/Makefile.vcs need this change?  Did you check that FPGA-level sim works with this bump?